vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/PriorityByWorkflow.java

```diff
@@ -53,17 +65,11 @@ public boolean remove(Object o) {
         return true;
       }
 
-      public SimpleEntry getByKey(String simpleEntryKey){
-        for (SimpleEntry entry : this){
-          if (entry.getKey().equals(simpleEntryKey)) {return(entry);}
-        }
-        return null;
-      }
     };
   }
 
 
-  //remove?
+  //not currently monitored by anything - potentially remove?
```

Comment:
We have lots of metrics that aren't monitored, and that's fine - they're for having around if something goes awry.